pkg/util/namer/namer_test.go

```diff
@@ -24,7 +24,7 @@ func TestGetName(t *testing.T) {
 			{
 				base:     longName,
 				suffix:   "deploy",
-				expected: longName[:244] + "-" + hash(longName+"-deploy"),
+				expected: longName[:237] + "-" + hash(longName) + "-deploy",
```

Comment:
My fault entirely... but since you're changing it :) ... it'd be clearer if the expected name uses util.DNS1123SubdomainMaxLength - some offset instead of a fixed number.